### PR TITLE
Upgrade to DS v42

### PR DESCRIPTION
**Pull request by rmccar**
### What is the context of this PR?
Upgrade to Design System to version 42.0.0

Fixes breaking changes in buttons and panels, removes 3 panel imports that are not being used and removes js-btn-save which isn't actually being set because its getting overwritten in header.

### How to review 
- Tests pass

### Checklist

* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

**Comment by rmccar** [comment]
[Update EQ Runner design-system version to 42.0.0 (3 SP)](https://trello.com/c/Vf2vw2d2/4910-update-eq-runner-design-system-version-to-4200-3-sp)

File: templates/census-thank-you.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
   {% call onsPanel({
     "type": "success",
     "classes": "ons-u-mb-s",
-    "icon": "check",
+    "iconType": "check",
```
Comment (MebinAbraham) [inline templates/census-thank-you.html:16]: There are other uses of `icon` such as on the hub that need updating. Currently, the icons on the hub is broken. 

Jinja filters also have references to the old way which needs updating.

File: templates/layouts/configs/_save-sign-out-button.html [hunk 2]
```diff
@@ -17,7 +17,8 @@
                 "data-ga-category": "Navigation",
                 "data-ga-action": "Save and sign out click"
             },
-            "buttonStyle": "exit"
```
Comment (MebinAbraham) [inline templates/layouts/configs/_save-sign-out-button.html:20]: Is `buttomStyle` still a valid use in the DS?

Comment (rmccar) [reply templates/layouts/configs/_save-sign-out-button.html:20]: It is for some button types but not for exit anymore. For exit we just add the icon, icon position and variant now

**Pull request merged**